packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderContextValues.ts

```diff
@@ -2,13 +2,14 @@ import * as React from 'react';
 import type { FluentProviderContextValues, FluentProviderState } from './FluentProvider.types';
 
 export function useFluentProviderContextValues_unstable(state: FluentProviderState): FluentProviderContextValues {
-  const { root, dir, targetDocument, theme } = state;
+  const { root, dir, targetDocument, theme, overrides } = state;
```

Comment:
It might be a good idea to have `unstable_overrides` even there (I know that `useFluentProviderContextValues_unstable` contains `unstable`, but...)